care/facility/api/viewsets/patient.py

```diff
@@ -649,8 +651,34 @@ def perform_create(self, serializer):
             raise ValidationError(
                 {"patient": "Only active patients data can be updated"}
             )
-        return serializer.save(
+
+        instance = serializer.save(
             facility=patient.facility,
             patient=patient,
             created_by=self.request.user,
         )
+
+        message = {
+            "facility_id": str(patient.facility.external_id),
+            "patient_id": str(patient.external_id),
+            "from": "patient/doctor_notes/create",
+        }
+
+        NotificationGenerator(
+            event=Notification.Event.MESSAGE,
```

Comment:
Message was meant for admin to facility comms, lets not reuse that.

Reply:
I see, I've moved it to a new type `PUSH_MESSAGE`.